manifests/init.pp

```diff
@@ -28,6 +28,7 @@
   Array[Cron::User]    $users_deny         = [],
   Boolean              $manage_users_allow = false,
   Boolean              $manage_users_deny  = false,
+  String               $mode               = '0640',
```

Comment:
```suggestion
  String               $mode               = '0644',
```

Don't change the default :-)

Reply:
Even if it's considered more secure ?

Reply:
Changing the default would be a breaking change, that's probably not necessary: these files contain name of users, not sensitive data, so hiding it's content is basically just security through obscurity, i.e. not security at all according to some :-)